client/src/components/common/Menu/index.css

```diff
@@ -4,7 +4,7 @@
   width: 50%;
   height: 100vh;
   right: 0;
-  margin-top: 58px;
+  margin-top: 0px;
```

Comment:
remove this line

Reply:
margin-top: 58px ; its already removed from my css file and replaced by margin-top:0px;